src/components/Quiz/Quiz.test.tsx

```
@@ -223,6 +228,28 @@
       <span>{label}</span>
     </button>
   ),
+  CardResults: ({
+    header,
+    correct_answers,
+    incorrect_answers,
+    icon,
+    direction,
```

ESLint

[error] 'direction' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)